neuro_flow/batch_runner.py

```diff
@@ -106,47 +186,77 @@ async def bake(self, batch_name: str) -> None:
         # check fast for the graph cycle error
         toposorter.prepare()
 
-        click.echo("Config is correct")
+        configs = [config_file, *await self._collect_additional_configs(flow)]
+        configs_data = [
+            (
+                RemotePath(path.relative_to(self._config_dir.workspace)),
+                path.read_text(),
+            )
+            for path in configs
+        ]
 
-        config_content = config_file.read_text()
+        click.echo("Config is correct")
 
         click.echo("Create bake")
         bake = await self._storage.create_bake(
             self.project,
             batch_name,
             config_file.name,
-            config_content,
+            configs_data,
         )
         click.echo(f"Bake {bake} created")
-        # TODO: run this function in a job
-        await self.process(bake.project, bake.batch, bake.when, bake.suffix)
+
+        data = ExecutorData(
+            project=bake.project,
+            batch=bake.batch,
+            when=bake.when,
+            suffix=bake.suffix,
+        )
+        if local_executor:
+            click.echo(f"Using local executor")
+            await self.process(data)
+        else:
+            click.echo(f"Starting remove executor")
+            param = data.serialize()
+            await self.run_subproc(
+                "neuro",
+                "run",
+                "--pass-config",
```

Comment:
please add `--restart=on-failure`

Reply:
Done. Should we also set `--life-span` for some value greater than 1 day?

Reply:
Good idea. Select something reasonable default.
Ideally, we should support both workflow life-span and concurrency in batch yaml.
It could be a separate PR, sure.